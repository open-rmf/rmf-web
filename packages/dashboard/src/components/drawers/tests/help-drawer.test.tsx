import React from 'react';
import { render } from '@testing-library/react';
import HelpDrawer from '../help-drawer';

test('Renders correctly', () => {
  const root = render(
    <HelpDrawer
      handleCloseButton={() => jest.fn()}
      setShowHotkeyDialog={() => jest.fn()}
      showTour={() => jest.fn()}
    />,
  );
  root.unmount();
});
