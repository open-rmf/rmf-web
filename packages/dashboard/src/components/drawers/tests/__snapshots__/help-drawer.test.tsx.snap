// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<WithStyles(ForwardRef(Drawer))
  PaperProps={
    Object {
      "className": "makeStyles-drawer-3",
    }
  }
  anchor="right"
>
  <DrawerHeader
    handleCloseButton={[Function]}
    title="Help"
  />
  <div
    className="makeStyles-detail-2"
  >
    <div
      className="makeStyles-detailLine-1"
    >
      <WithStyles(ForwardRef(IconButton))
        color="inherit"
        id="show-manual-btn"
      >
        <MenuBookIcon />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Typography))
        variant="h5"
      >
         Tutorial 
      </WithStyles(ForwardRef(Typography))>
    </div>
    <WithStyles(ForwardRef(Divider)) />
    <div
      className="makeStyles-detailLine-1"
      onClick={[Function]}
    >
      <WithStyles(ForwardRef(IconButton))
        color="inherit"
        id="show-hotkeys-btn"
      >
        <DirectionsIcon />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Typography))
        variant="h5"
      >
         Hotkeys 
      </WithStyles(ForwardRef(Typography))>
    </div>
    <WithStyles(ForwardRef(Divider)) />
    <div
      className="makeStyles-detailLine-1"
    >
      <WithStyles(ForwardRef(IconButton))
        color="inherit"
        id="show-hotkeys-btn"
      >
        <BugReportIcon />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Typography))
        variant="h5"
      >
         Report an error 
      </WithStyles(ForwardRef(Typography))>
    </div>
    <WithStyles(ForwardRef(Divider)) />
  </div>
</WithStyles(ForwardRef(Drawer))>
`;
